src/redux/actions/samples/deleteSamples.js

```diff
@@ -87,26 +105,36 @@ const deleteSamples = (
         // This is set right now as there is only one experiment per project
         // Should be changed when we support multiple experiments per project
         const experimentId = projects[projectUuid].experiments[0];
-        await sendDeleteSamplesRequest(projectUuid, experimentId, sampleUuids);
-
-        dispatch(saveProject(projectUuid, newProject, false));
 
-        dispatch({
-          type: PROJECTS_UPDATE,
-          payload: {
-            projectUuid,
-            project: {
-              samples: newSamples,
-            },
-          },
-        });
+        if (config.currentApiVersion === api.V1) {
+          await sendDeleteSamplesRequest(projectUuid, experimentId, sampleUuids);
 
-        dispatch({
-          type: SAMPLES_DELETE,
-          payload: { sampleUuids: samplesToDelete },
-        });
+          dispatch(saveProject(projectUuid, newProject, false));
 
-        dispatch(updateExperiment(experimentId, { sampleIds: newSamples }));
+          dispatch({
+            type: PROJECTS_UPDATE,
+            payload: {
+              projectUuid,
+              project: {
+                samples: newSamples,
+              },
+            },
+          });
+
```

Comment:
why do you not need to dispatch `PROJECTS_UPDATE` in v2?

Reply:
becuase I added instead a reducer in the `projects` slice for `SAMPLES_DELETE`, same thing for `experiments` slice and `samples` slice